### PR TITLE
D9 Ready

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- removed inherited dependencies.
- Disabled module that we can do our self without relying on a poorly maintained module.

# Need Review By (Date)
- end of the year

# Urgency
- low

# Steps to Test

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by pookmish** [comment]
Ok I believe I'm done editing the tests. They are passing on D8 with the test run here and they are also passing on D9 setup https://app.circleci.com/pipelines/github/SU-SWS/acsf-cardinalsitesd9/16/workflows/e2176e20-9b6d-4a93-a2ca-b201253d34bf/jobs/55

File: config/sync/core.extension.yml [hunk 1]
```diff
@@ -15,7 +15,6 @@ module:
   ckeditor: 0
   ckeditor_a11ychecker: 0
   ckeditor_balloonpanel: 0
-  ckeditor_fixed_toolbar: 0
```
Comment (sherakama) [inline config/sync/core.extension.yml:18]: Where is this functionality going?

File: tests/codeception/acceptance/LocalFooter/LocalFooterLockupCest.php [hunk 2]
```diff
@@ -40,22 +43,21 @@ function _before(AcceptanceTester $I) {
...
     $I->amOnPage('/admin/config/system/local-footer');
-    $I->uncheckOption('#edit-su-local-foot-use-loc-value');
-    $I->uncheckOption('#edit-su-local-foot-use-logo-value');
-    $I->selectOption("#edit-su-local-foot-loc-op", "a");
+    $I->uncheckOption('Use Default Lockup');
+    $I->uncheckOption('Use the logo supplied by the theme');
+    $I->selectOption('Lockup Options', "a");
     // In case there was an image already.
-    try {
+    if ($I->grabMultiple('input[value="Remove"]')) {
```
Comment (sherakama) [inline tests/codeception/acceptance/LocalFooter/LocalFooterLockupCest.php:50]: That is much tidier. Thanks.

File: tests/codeception/acceptance/LocalFooter/LocalFooterLockupCest.php [hunk 3]
```diff
@@ -197,14 +199,16 @@ public function testLockupSettingsI(AcceptanceTester $I) {
 
    /**
    * Test the lockup settings overrides.
+    *
+    * @group testme
```
Comment (sherakama) [inline tests/codeception/acceptance/LocalFooter/LocalFooterLockupCest.php:203]: Remove group.

File: tests/codeception/acceptance/LocalFooter/LocalFooterLockupCest.php [hunk 4]
```diff
@@ -28,6 +28,9 @@ class LocalFooterLockupCest {
   function _before(AcceptanceTester $I) {
     $this->DATA_DIR = rtrim(codecept_data_dir(), '/\\');
     // Copy our assets into place first.
+    if (!file_exists($this->DATA_DIR . DIRECTORY_SEPARATOR)) {
```
Comment (sherakama) [inline tests/codeception/acceptance/LocalFooter/LocalFooterLockupCest.php:31]: Shouldn't this be `is_dir`?

Comment (pookmish) [reply tests/codeception/acceptance/LocalFooter/LocalFooterLockupCest.php:31]: https://www.php.net/manual/en/function.file-exists.php
> Checks whether a file or directory exists.

Tomato Potato.

File: config/sync/core.extension.yml (hunk 1 above)
Comment (pookmish) [reply config/sync/core.extension.yml:18]: It doesn't even work at the moment from what I found. I'm thinking it'll be better suited for stanford_profile_helper https://stanfordits.atlassian.net/browse/D8CORE-1420

File: tests/codeception/acceptance/LocalFooter/LocalFooterLockupCest.php (hunk 2 above)
Comment (pookmish) [reply tests/codeception/acceptance/LocalFooter/LocalFooterLockupCest.php:50]: FWIW the try catch didn't work in the D9 testing. It wouldn't ever `catch`. It just failed.

Comment (sherakama) [reply tests/codeception/acceptance/LocalFooter/LocalFooterLockupCest.php:50]: Most interesting.

File: config/sync/core.extension.yml (hunk 1 above)
Comment (sherakama) [reply config/sync/core.extension.yml:18]: It was working for me on `/node/add/stanford_event`

Comment (sherakama) [reply config/sync/core.extension.yml:18]: https://devguide.sites.stanford.edu/node/add/stanford_event

File: tests/codeception/acceptance/LocalFooter/LocalFooterLockupCest.php (hunk 4 above)
Comment (sherakama) [reply tests/codeception/acceptance/LocalFooter/LocalFooterLockupCest.php:31]: WOrDs MeAN THingS!

File: config/sync/core.extension.yml (hunk 1 above)
Comment (pookmish) [reply config/sync/core.extension.yml:18]: Hmm. Maybe just the basic page its not working then. I guess the question is, do we do the fork thing in drupal.org to support a non-stable module, or do we move the work into the profile helper module.

Comment (sherakama) [reply config/sync/core.extension.yml:18]: I'm not familiar with the issues this module is causing but for what functionality it does offer, I suspect we can do without the trouble. Sounds like you have some custom work to do to get it to work in the react modal.

**Review by sherakama — APPROVED** [review]
GTG when something like https://github.com/SU-SWS/stanford_profile_helper/pull/52 gets completed.

**Comment by pookmish** [comment]
I was just wrapping up the tests https://github.com/SU-SWS/stanford_text_editor/pull/42

**Pull request merged**